cmd/template/cluster/flag.go

```diff
@@ -78,6 +84,24 @@ func (f *flag) Init(cmd *cobra.Command) {
 func (f *flag) Validate() error {
 	var err error
 
+	if f.ClusterID != "" {
+		if len(f.ClusterID) != key.IDLength {
+			return microerror.Maskf(invalidFlagError, "--%s must be length of %d", flagClusterID, key.IDLength)
+		}
+
+		matched, err := regexp.MatchString("^([a-z]+|[0-9]+)$", f.ClusterID)
+		if err == nil && matched == true {
+			// strings is letters only, which we also avoid
```

Comment:
This should be "strings is letters only or numbers only", which we avoid for usability reasons